src/ray/object_manager/transfer_queue.h

```diff
@@ -0,0 +1,115 @@
+#ifndef RAY_TRANSFER_QUEUE_H
+#define RAY_TRANSFER_QUEUE_H
+
+#include <algorithm>
+#include <cstdint>
+#include <deque>
+#include <map>
+#include <memory>
+#include <thread>
+
+#include <boost/asio.hpp>
+#include <boost/asio/error.hpp>
+#include <boost/bind.hpp>
+
+#include "ray/id.h"
+#include "ray/status.h"
+
+#include "format/object_manager_generated.h"
+#include "object_directory.h"
+#include "object_manager_client_connection.h"
+
+namespace asio = boost::asio;
```

Comment:
Why do this? It doesn't seem to be used. Also we use `boost::asio` everywhere else.

Reply:
Removed; I do use it in other places though (and it's a permissible based on Google c-style guide).

Reply:
I agree, short namespace prefixes should just be used directly. Using the `using` directive makes sense in places where it's one-line long callback specifications, etc.